stable/fairwinds-insights/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "8.1.1"
 description: A Helm chart to run the Fairwinds Insights server
 name: fairwinds-insights
-version: 0.4.4
+version: 0.4.6
```

Comment:
Why not `0.4.5`?

Reply:
Indeed, I was sure I got some changes on master that made me bump this again from 0.4.5 to 0.4.6